op-dispute-mon/mon/bonds/monitor.go

```diff
@@ -47,7 +47,7 @@ func (b *Bonds) CheckBonds(games []*types.EnrichedGameData) {
 func (b *Bonds) checkCredits(game *types.EnrichedGameData) {
 	// Check if the max duration has been reached for this game
 	duration := uint64(b.clock.Now().Unix()) - game.Timestamp
-	maxDurationReached := duration >= game.Duration
+	maxDurationReached := duration >= game.MaxClockDuration
```

Comment:
This should be at least twice the max clock duration. @refcell would it be more accurate to check if the root clock has expired rather than use `maxDurationReached`?

Reply:
Hmm - good q. There's really no "max duration" anymore, without deriving it from the configured depth and the possible clock extension upper bound. We really want to see when the clocks expire.